FEM/fem_ele.h

```diff
@@ -170,18 +217,34 @@ class CElement
 	int Order;
 	size_t ele_dim; // Dimension of element
 	size_t dim; // Dimension of real dimension
+	size_t dim_grad; // Gradient dimension
```

Comment:
i am not sure what is "gradient dimension". Is it different from the spacial dimension?